### PR TITLE
Community demos - Iris Classification and linear regression

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: demos_community.rst [hunk 1]
```diff
@@ -70,6 +70,60 @@ Community
     show how to characterize the loss landscape with it. We show how the Hessian can be used to
     escape flat regions of the loss landscape.
 
+.. community-card::
+    :title: Iris Classification using Angle Embedding and qml.qnn.KerasLayer
+    :author: Hemant Gahankari
+    :date: 09/11/2020
```
Comment (antalszava) [inline demos_community.rst:76]: Going with the date for acceptance of the submission (could update to the day of merging if preferred)

**Comment by co9olguy** [comment]
Thanks @antalszava!

File: demos_community.rst [hunk 2]
```diff
@@ -70,6 +70,60 @@ Community
...
+    :code: https://colab.research.google.com/drive/1ABVtBjwcGNNIfmiwEXRdFdZ47K1vZ978?usp=sharing
+    :color: heavy-rain-gradient
+
+    This example is created to explain how to create a Hybrid Neural Network
+    (Mix of Classical and Quantum Layer) and train it and get predictions from
+    it. The data set consists of a temperature readings in Degree Centigrade
+    and corresponding Fahrenheit. Objective is to train a neural network to
+    predict Fahrenheit value given Deg Centigrade value.
```
Comment (thisac) [inline demos_community.rst:110]: Again, maybe we shouldn't care about grammar/wording here.

```suggestion
    This example is created to explain how to create a Hybrid Neural Network
    (mix of classical and quantum layers) and train it and get predictions from
    it. The data set consists of temperature readings in degrees Centigrade
    and corresponding Fahrenheit. The objective is to train a neural network to
    predict Fahrenheit values given Centigrade values.
```

File: demos_community.rst [hunk 3]
```diff
@@ -70,6 +70,60 @@ Community
...
+    :title: Iris Classification using Qnode and Keras Optimizer and Loss function
+    :author: Hemant Gahankari
+    :date: 09/11/2020
+    :code: https://colab.research.google.com/drive/17Qri3jUBpjjkhmO6ZZZNXwm511svSVPw?usp=sharing
+    :color: heavy-rain-gradient
+
+    This example is created to explain how to create a quantum function and
+    train a quantum function using keras optimizer directly. i.e. not using
+    Keras Layer. Objective is to train a quantum function to predict classes of
+    Iris dataset.
```
Comment (thisac) [inline demos_community.rst:125]:
```suggestion
    This example is created to explain how to create a quantum function and
    train a quantum function using Keras optimizer directly, i.e., not using
    a Keras layer. The objective is to train a quantum function to predict classes of
    the Iris dataset.
```

**Review by thisac — COMMENTED** [review]
Thanks @antalszava for adding these. :tada:  I just have a few comments/thought on how we should handle the text/grammar in the abstracts. Since they are hosted on our page, should we check spelling/grammar here. I guess Josh's guidelines could mean both: 

_We should:
Ensure the card content the submitter provides is up to our standards (title, abstract, etc.) since it is on our website (this may require some back and forth on the issue)._

_We should not: 
Check spelling and grammar_ <-- **I take this as in the actual demos/code.**

File: demos_community.rst [hunk 4]
```diff
@@ -70,6 +70,60 @@ Community
...
+.. community-card::
+    :title: Iris Classification using Angle Embedding and qml.qnn.KerasLayer
+    :author: Hemant Gahankari
+    :date: 09/11/2020
+    :code: https://colab.research.google.com/drive/13PvS2D8mxBvlNw6_5EapUU2ePKdf_K53#scrollTo=1fJWDX5LxfvB
+    :color: heavy-rain-gradient
+
+    This example is created to explain how to pass classical data into the quantum
+    function and convert it to quantum data. Later it shows how to create
+    qml.qnn.KerasLayer from qnode and train it and also check model performance.
```
[the remark this answers is not in the document]
Comment (antalszava) [reply demos_community.rst:82]: Good call, thanks!

**Review by thisac — APPROVED** [review]
Looks great @antalszava! Nice to get this page populated with more demos. :100:

**Review by trbromley — APPROVED** [review]
Thanks @antalszava!

**Pull request merged**